pkgs/apps/exatensor/default.nix

```
@@ -0,0 +1,54 @@
+{ stdenv, lib, fetchFromGitLab, blas, mpi } :
+
+let
+  version = "73fef4";
+
+in stdenv.mkDerivation rec {
+  pname = "exatensor";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitLab {
+    owner = "DmitryLyakh";
+    repo = pname;
+    rev = "d304c03b7289525f250245ece1b4a13a5973fef4";
+    sha256 = "0zlqwpdqfycwvys39lvvj94cprhaxmyaxxs1c66f10dazjbafp49";
+  };
+
+  # nativeBuildInputs = [ make ];
```

Basically never needed with `stdenv`
```suggestion
-  # nativeBuildInputs = [ make ];
```